compiler-lib/compiler/commands/test.rkt

```diff
@@ -666,7 +666,13 @@
   ;; This should be in Racket somewhere and return all the collection
   ;; paths, rather than just the first as collection-path does.
   (define (collection-paths c)
-    (match-define (list-rest sc more) (map path->string (explode-path c)))
+    (define (path->string* maybe-path)
```

Comment:
Yes, that's what I meant. Basically I think that path->string should be doing this. And if (p->s (b-p ....)) does it, then we should use that.